ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -963,6 +963,16 @@ def uninstall_master(host, ignore_topology_disconnect=True,
 
     result = host.run_command(uninstall_cmd)
     assert "Traceback" not in result.stdout_text
+
+    # Check that IPA certs have been deleted after uninstall
+    # Related: https://pagure.io/freeipa/issue/8614
+    assert host.run_command(['test', '-f', paths.IPA_CA_CRT],
+                            raiseonerr=False).returncode == 1
+    assert host.run_command(['test', '-f', paths.IPA_P11_KIT],
+                            raiseonerr=False).returncode == 1
+    assert "IPA CA" not in host.run_command(['trust', 'list'],
+                                            raiseonerr=False).stdout_text
```

Comment:
If this command fails this assert will still pass. Dropping the raiseonerr=False will address this. Or you can assign the result to a variable and assert returncode == 0 along with the 'IPA CA' not in stdout_text.